splunklib/searchcommands/generating_command.py

```diff
@@ -174,6 +174,11 @@ class SomeCommand(GeneratingCommand)
         streaming = false
 
     """
+
+    # overwrite reply_message_v2
+
+    reply_message_v2=(('type', "streaming"),("generating",True))
```

Comment:
I'm not sure I follow this. How would I make a GeneratingCommand that is stateful (i.e. runs on the search head) or reporting (i.e. produces results in the Statistics tab and not the Events tab in the S&R UI?)?